src/pages/sighting/dialogs/ReviewSightingDialog.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+
+import useReviewSighting from '../../../models/sighting/useReviewSighting';
+import StandardDialog from '../../../components/StandardDialog';
+import Button from '../../../components/Button';
+import Text from '../../../components/Text';
+import Alert from '../../../components/Alert';
+
+export default function SightingHistoryDialog({
```

Comment:
If we have the chance to name the component that same thing as the filename, we should probably do that.